labman/gui/handlers/auth.py

```diff
@@ -29,6 +31,8 @@ def post(self):
             error_msg = "Unknown user name"
         except LabmanLoginError:
             error_msg = "Incorrect password"
+        except LabmanLoginDisabledError:
+            error_msg = "User not allowed on this portal"
```

Comment:
There are multiple portals into labman, or just that the user doesn't have access to labman?